nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -28,18 +28,24 @@ define([
 
     const div = html.tag('div');
 
-    class DefaultWidget {
-        constructor() {
-
-        }
+    function DefaultWidget() {
+        function make() {
+            function start(options) {
```

Comment:
Should we remove options until it is used?

Reply:
Really that whole `DefaultWidget` is just there as a placeholder. I'll remove the options bit.